### PR TITLE
Update kbs-tools README.md

### PR DESCRIPTION
**Pull request by EmmEff**
Reflect recent project restructuring.

### CODE REVIEW

**Review by Xynnn007 — APPROVED** [review]
Thanks for this!

File: tools/kbs-client/README.md [hunk 1]
```diff
@@ -32,13 +32,13 @@ require fewer dependencies and so usually handy for CI) then you can pass the
 `sample_only` feature as:
 
 ```shell
-make -C ../../ cli CLI_FEATURES=sample_only
+make -C ../../kbs cli CLI_FEATURES=sample_only
```
Comment (mythi) [inline tools/kbs-client/README.md:35]: the change itself is OK but the comment above about `sample_only` looks outdated so we could update that too while we're at it. 

Default features come with `sample_only`. This section here should probably give instructions how to build a TEE specific flavor with: `make -C ../../kbs cli ATTESTER=<selected attester, e.g., tdx-attester`.

Comment (fitzthum) [reply tools/kbs-client/README.md:35]: The command in the diff is correct. Describing how to build with different attesters seems beyond the scope of this PR even though it's a pretty simple change.

**Review by fitzthum — APPROVED** [review]
LGTM

File: tools/kbs-client/README.md (hunk 1 above)
Comment (mythi) [reply tools/kbs-client/README.md:35]: Indeed. I did not check the config before adding my comment. My recollection was that it was sample-only by default.

**Pull request merged**